internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -609,6 +609,18 @@ func convertEntryPoint(entrypoint manifest.EntryPointOverride) ([]string, error)
 	return out, nil
 }
 
+func getDeployConfig(deployStrategyName string) template.DeploymentConfigurationOpts {
```

Comment:
what do you think about changing the arg's type to `manifest.DeploymentConfiguration`?

Reply:
But `manifest.DeploymentConfiguration` type is different than  `template.DeploymentConfigurationOpts`

Reply:
sorry, i wasn't very clear. i meant the input arg, the same idea @Lou1415926 mentioned [here](https://github.com/aws/copilot-cli/pull/3461#discussion_r849840841) :)